src/Azs.Compute.Admin/docs/Update-AzsGlobalFeatureSetting.md

```diff
@@ -0,0 +1,262 @@
+---
+external help file:
+Module Name: Azs.Compute.Admin
+online version: https://docs.microsoft.com/en-us/powershell/module/azs.compute.admin/update-azsglobalfeaturesetting
+schema: 2.0.0
+---
+
+# Update-AzsGlobalFeatureSetting
```

Comment:
Is this a PATCH call or PUT. FOr PUT calls, SET-* verb is preferred

Reply:
We looked at this and it is a POST.

Reply:
Typical Post call should be called with Invoke-* here Invoke-AzsUpdateGlobalFeatureSetting . i dont think that naming make sense here, lets keep it as such